src/ncp/ncp_base.hpp

```diff
@@ -266,6 +266,9 @@ class NcpBase
 #if OPENTHREAD_FTD
     static void HandleChildTableChanged(otThreadChildTableEvent aEvent, const otChildInfo *aChildInfo);
     void        HandleChildTableChanged(otThreadChildTableEvent aEvent, const otChildInfo &aChildInfo);
+
+    static void HandleParentResponseInfo(otThreadParentResponseInfo *aInfo, void *aContext);
```

Comment:
HandleParentResponseReceived (to keep with the naming convention of other handlers)

Reply:
I'm fine either way. Will defer to @abtink / @jwhui